py/server/deephaven/parquet.py

```diff
@@ -449,6 +466,11 @@ def batch_write(
     Raises:
         DHError
     """
+    if col_definitions is not None:
+        warn("col_definitions is deprecated and will be removed in a future release. Use table_definition "
```

Comment:
Do we have a ticket for doing the API clean up?  There _may_ be a DHE drive for the removal to hit a major release.  Talk to @cpwright about timing.  I have a few clean ups like this removal that are waiting to get merged.

Reply:
I have added the ticket for Parquet refactoring. Will co-ordinate with Charles and the enterprise team before the breaking changes.